extensions/tekton/model/src/main/java/io/fabric8/tekton/TektonResourceMappingProvider.java

```diff
@@ -38,15 +38,19 @@ public TektonResourceMappingProvider() {
         mappings.put("tekton.dev/v1alpha1#PipelineResource", PipelineResource.class);
         mappings.put("tekton.dev/v1alpha1#Condition", Condition.class);
         mappings.put("tekton.dev/v1beta1#Pipeline", Pipeline.class);
```

Comment:
nit: In order to make it clearer, I thin it would be better to remove global imports and add the canonical path to all classes here:
```suggestion
        mappings.put("tekton.dev/v1beta1#Pipeline", io.fabric8.tekton.pipeline.v1beta1.Pipeline.class);
```

Reply:
+1